kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -0,0 +1,656 @@
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Default constructor. (with parameters)
+     * @param rModelPart The model part of the problem
+     * @param ThisParameters The configuration parameters
+     */
+    explicit ExplicitSolvingStrategy(
+        ModelPart &rModelPart,
+        Parameters ThisParameters)
+        : mrModelPart(rModelPart)
+    {
+        const bool rebuild_level = ThisParameters.Has("rebuild_level") ? ThisParameters["rebuild_level"].GetInt() : 0;
+        const bool move_mesh_flag = ThisParameters.Has("move_mesh_flag") ? ThisParameters["move_mesh_flag"].GetBool() : false;
+        SetMoveMeshFlag(move_mesh_flag);
+        SetRebuildLevel(rebuild_level);
+        mpExplicitBuilder = Kratos::make_unique<ExplicitBuilder<TSparseSpace, TDenseSpace>>();
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param pExplicitBuilder The pointer to the explicit builder and solver
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     * @param RebuildLevel The flag to set if the DOF set is rebuild or not
+     */
+    explicit ExplicitSolvingStrategy(
+        ModelPart &rModelPart,
+        typename ExplicitBuilderType::Pointer pExplicitBuilder,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : mrModelPart(rModelPart),
+          mpExplicitBuilder(pExplicitBuilder)
+    {
+        SetMoveMeshFlag(MoveMeshFlag);
+        SetRebuildLevel(RebuildLevel);
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     * @param RebuildLevel The flag to set if the DOF set is rebuild or not
+     */
+    explicit ExplicitSolvingStrategy(
+        ModelPart &rModelPart,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : mrModelPart(rModelPart)
+    {
+        SetMoveMeshFlag(MoveMeshFlag);
+        SetRebuildLevel(RebuildLevel);
+        mpExplicitBuilder = Kratos::make_unique<ExplicitBuilder<TSparseSpace, TDenseSpace>>();
+    }
+
+    /** Copy constructor.
+     */
+    ExplicitSolvingStrategy(const ExplicitSolvingStrategy &Other) = delete;
+
+    /** Destructor.
+     */
+    virtual ~ExplicitSolvingStrategy() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /**
+     * @brief Operation to predict the solution ... if it is not called a trivial predictor is used in which the
+     * values of the solution step of interest are assumed equal to the old values
+     */
+    virtual void Predict()
+    {
+    }
+
+    /**
+     * @brief Initialization of member variables and prior operations
+     */
+    virtual void Initialize()
+    {
+        // Initialize elements, conditions and constraints
+        InitializeContainer(GetModelPart().Elements());
```

Comment:
Entities utilities?

Reply:
we already discussed this, the EntitiesUtilities only initializes the active ones for some reason

Reply:
OK then, I had other commentary anyway